server/tables.sql

```diff
@@ -1,11 +1,18 @@
 USE LIHI;
 CREATE TABLE IF NOT EXISTS VILLAGES (VID INT PRIMARY KEY auto_increment, NAME VARCHAR(50) NOT NULL UNIQUE);
-CREATE TABLE IF NOT EXISTS PERMISSIONS (ROLE_ID INT PRIMARY KEY auto_increment, DESCRIPTION VARCHAR(255) NOT NULL UNIQUE);
-CREATE TABLE IF NOT EXISTS PEOPLE (PID INT PRIMARY KEY auto_increment, NAME VARCHAR(50) NOT NULL, ROLE_ID INT NOT NULL, VID INT NOT NULL, CONSTRAINT FK_Permission FOREIGN KEY (ROLE_ID) REFERENCES PERMISSIONS(ROLE_ID), CONSTRAINT FK_Village FOREIGN KEY (VID) REFERENCES VILLAGES(VID));
+CREATE TABLE IF NOT EXISTS PERMISSIONS (ROLE_ID INT PRIMARY KEY auto_increment, ROLE_TYPE VARCHAR(255) NOT NULL UNIQUE);
+CREATE TABLE IF NOT EXISTS PEOPLE (PID INT PRIMARY KEY auto_increment, FIRST_NAME VARCHAR(50) NOT NULL, LAST_NAME VARCHAR(50) NOT NULL, BIRTHDAY DATE, ROLE_ID INT NOT NULL, VID INT NULL, CONSTRAINT FK_Permission FOREIGN KEY (ROLE_ID) REFERENCES PERMISSIONS(ROLE_ID), CONSTRAINT FK_Village FOREIGN KEY (VID) REFERENCES VILLAGES(VID));
 CREATE TABLE IF NOT EXISTS ROOMS (ROOM_ID INT PRIMARY KEY auto_increment, ROOM_NUM INT NOT NULL, BLDG_NAME VARCHAR(50) NOT NULL, VID INT NOT NULL, VACANT BOOLEAN, CONSTRAINT FK_VID FOREIGN KEY (VID) REFERENCES VILLAGES(VID));
```

Comment:
This needs to be updated as we just have house numbers and there are no buildings.

Its just village and house.

Reply:
Fixed.